src/ui/score_ui.cpp

```diff
@@ -63,13 +63,33 @@ ScoreUI::ScoreUI(float aspect)
     events::ui::scoreboard_sequence.connect([&](const auto &data) {
         sorted_data = data;
         populate_scores();
-        enable();
+        enable_animated();
 
         // Do gold delta sequence.
         animate_deltas();
     });
 
     events::ui::disable_scoreboard.connect([&]() {
+        disable_animated();
+    });
+}
+
+void ScoreUI::enable_animated() {
```

Comment:
well different UIs animate in different ways. the base UI does have a default thingy though.